R/getGate.R

```diff
@@ -0,0 +1,20 @@
+#' Get a gate
+#'
+#' Retrieves the list of gates in an experiment.
+#'
+#' @param experimentId ID of experiment.
+#' @param params Optional query parameters.
+#' @export
+#' @examples
+#' \dontrun{
+#' getGate(experimentId, gateId)
+#' getGate(experimentId, gateId, params = list("limit" = "5"))
```

Comment:
You can remove the `params` arg (the API takes non).

Reply:
right, only for _list_ routes